packages/web/src/scss/components/Item/index.html

```diff
@@ -1,5 +1,120 @@
 {{#> layout/plain }}
 
+<section class="docs-Section">
+  <h2 class="docs-Heading">Default</h2>
+  <div class="docs-Stack docs-Stack--stretch">
+
+    <div class="Item">
+      <span class="Item__label">Item label</span>
+    </div>
+
+    <a href="https://www.example.com" class="Item">
+      <span class="Item__label">Item label</span>
+    </a>
+
+  </div>
+</section>
+
+<section class="docs-Section">
+  <h2 class="docs-Heading">Selected</h2>
+  <div class="docs-Stack docs-Stack--stretch">
+
+    <div class="Item Item--selected">
+      <span class="Item__label">Item label</span>
+      <span class="Item__icon Item__icon--end">
+        <svg width="24" height="24" aria-hidden="true">
+          <use xlink:href="/icons/svg/sprite.svg#check-plain" />
+        </svg>
+      </span>
+    </div>
+
+  </div>
+</section>
+
+<section class="docs-Section">
+  <h2 class="docs-Heading">Disabled</h2>
+  <div class="docs-Stack docs-Stack--stretch">
+
+    <div class="Item Item--disabled">
+      <span class="Item__label">Item label</span>
+    </div>
+
+  </div>
+</section>
+
+<section class="docs-Section">
+  <h2 class="docs-Heading">Default</h2>
```

Comment:
Should this title be `Default`?

Reply:
Probably not, it's mostly about helper text…